flink-python/src/test/java/org/apache/flink/client/python/PythonDependencyUtilsTest.java

```diff
@@ -0,0 +1,246 @@
+			"tmp_dir/requirements2.txt#tmp_dir/cache");
+		actual = PythonDependencyUtils.configurePythonDependencies(cachedFiles, config);
+
+		expectedCachedFiles = new HashMap<>();
+		expectedCachedFiles.put(
+			"python_requirements_file_56fd0c530faaa7129dca8d314cf69cbfc7c1c5c952f5176a003253e2f418873e",
+			"tmp_dir/requirements2.txt");
+		expectedCachedFiles.put(
+			"python_requirements_cache_2f563dd6731c2c7c5e1ef1ef8279f61e907dc3bfc698adb71b109e43ed93e143",
+			"tmp_dir/cache");
+		verifyCachedFiles(expectedCachedFiles);
+
+		expectedConfiguration = new Configuration(config);
+		expectedConfiguration.set(PYTHON_REQUIREMENTS_FILE, new HashMap<>());
+		expectedConfiguration.get(PYTHON_REQUIREMENTS_FILE).put(
+			"file",
+			"python_requirements_file_56fd0c530faaa7129dca8d314cf69cbfc7c1c5c952f5176a003253e2f418873e");
+		expectedConfiguration.get(PYTHON_REQUIREMENTS_FILE).put(
+			"cache",
+			"python_requirements_cache_2f563dd6731c2c7c5e1ef1ef8279f61e907dc3bfc698adb71b109e43ed93e143");
+		verifyConfiguration(expectedConfiguration, actual);
+	}
+
+	@Test
+	public void testAddPythonArchive() {
+		Configuration config = new Configuration();
+		config.set(
+			PythonOptions.PYTHON_ARCHIVES,
+			"hdfs:///tmp_dir/file1.zip," +
+			"hdfs:///tmp_dir/file1.zip," +
+			"tmp_dir/py37.zip," +
+			"tmp_dir/py37.zip#venv," +
+			"tmp_dir/py37.zip#venv2,tmp_dir/py37.zip#venv");
+		Configuration actual = PythonDependencyUtils.configurePythonDependencies(cachedFiles, config);
+
+		Map<String, String> expectedCachedFiles = new HashMap<>();
+		expectedCachedFiles.put(
+			"python_archive_4cc74e4003de886434723f351771df2a84f72531c52085acc0915e19d70df2ba",
+			"hdfs:///tmp_dir/file1.zip");
+		expectedCachedFiles.put(
+			"python_archive_b9ceeb8434d3335ec0ba298bbd52768bf5e4d136df3e157500fe4f413a1f416c",
+			"tmp_dir/py37.zip");
+		expectedCachedFiles.put(
+			"python_archive_5f3fca2a4165c7d9c94b00bfab956c15f14c41e9e03f6037c83eb61157fce09c",
+			"tmp_dir/py37.zip");
+		expectedCachedFiles.put(
+			"python_archive_44532e54ce38086fcc235a2af12479a7305820f7f39767f0215c454df54024dd",
+			"tmp_dir/py37.zip");
+		verifyCachedFiles(expectedCachedFiles);
+
+		Configuration expectedConfiguration = new Configuration(config);
+		expectedConfiguration.set(PYTHON_ARCHIVES, new HashMap<>());
+		expectedConfiguration.get(PYTHON_ARCHIVES).put(
+			"python_archive_4cc74e4003de886434723f351771df2a84f72531c52085acc0915e19d70df2ba", "file1.zip");
+		expectedConfiguration.get(PYTHON_ARCHIVES).put(
+			"python_archive_5f3fca2a4165c7d9c94b00bfab956c15f14c41e9e03f6037c83eb61157fce09c", "py37.zip");
+		expectedConfiguration.get(PYTHON_ARCHIVES).put(
+			"python_archive_b9ceeb8434d3335ec0ba298bbd52768bf5e4d136df3e157500fe4f413a1f416c", "venv");
+		expectedConfiguration.get(PYTHON_ARCHIVES).put(
+			"python_archive_44532e54ce38086fcc235a2af12479a7305820f7f39767f0215c454df54024dd", "venv2");
+		verifyConfiguration(expectedConfiguration, actual);
+	}
+
+	@Test
+	public void testConfigure() {
```

Comment:
It seems to me that this test case is redundant and can be removed. What do you think?